src/pages/User/User.layout.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <user-header
+      :expirationDate="expirationDate"
+      :isMembershipActive="isMembershipActive"
+      :avatar="avatar"
+    />
+    <div>
+      <user-data
+        :firstName="firstName"
+        :lastName="lastName"
+        :memberCode="memberCode"
+        :email="email"
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { UserType } from './mock';
```

Comment:
Please check this user-type, maybe you could use it instead of add a new User-Type

https://github.com/Azordev/did-app/blob/a4575c572fbd85ec44f3c83b2076c7200aa102e4/src/utils/apollo.types.ts#L3-L13

If you need more attributes in this type, feels free of modify it

Reply:
yes, I added member-info attribute